composer.json

```diff
@@ -11,36 +11,39 @@
     }
   },
   "require": {
-    "php": "^7.0",
+    "php": "^7.1.3",
     "ext-PDO": "*",
     "ext-json": "*",
     "ext-sockets": "*",
-    "composer/composer": "@stable",
+    "composer/composer": "^1.4",
     "composer/semver": "^1.4",
     "graylog2/gelf-php": "^1.4.2",
     "guzzlehttp/guzzle": "^6.2",
-    "illuminate/config": "^5.5",
-    "illuminate/container": "^5.5",
-    "illuminate/contracts": "^5.5",
+    "illuminate/config": "^5.6",
+    "illuminate/container": "^5.6",
+    "illuminate/contracts": "^5.6",
     "magento/magento-cloud-components": "^1.0.1",
-    "magento/magento-cloud-docker": "^1.0",
     "monolog/monolog": "^1.16",
     "psr/container": "^1.0",
     "psr/log": "^1.0",
-    "symfony/console": "^2.6||^4.0",
-    "symfony/process": "~2.1||~4.1.0",
-    "symfony/yaml": "^3.3||^4.0",
-    "twig/twig": "^1.0||^2.0"
+    "symfony/console": "^2.3||^4.0",
+    "symfony/process": "^2.1||^4.1",
+    "symfony/serializer": "^4.2",
+    "symfony/yaml": "^4.0",
+    "twig/twig": "^2.0"
```

Comment:
Do we really want to set hard dependencies for modules like twig which are used in ECE-Tools for a single mail function? Just worried that it may be incompatible with some extensions.

Reply:
It was required for many releases already, you can add task to remove twig

Reply:
@shiftedreality my concern is that you removed 1.x release line (not the fact that twig is here in general). I honestly think that adding twig is overkill for 1 email template but current question is not about that

Reply:
Got it. I reverted this change and agree that we can remove it completely. Please create a separate task.

Reply:
Created `MAGECLOUD-4272` as tech debt for it.